
From: Olaf Hering <olh@suse.de>

drivers/video/console/fbcon.c:310: warning: comparison is always true due
to limited range of data type

char can be either signed or unsigned, depending on the target system. 

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/drivers/video/console/fbcon.c |    2 +-
 25-akpm/drivers/video/console/fbcon.h |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff -puN drivers/video/console/fbcon.c~signed-bug-in-drivers-video-console-fbconc-con2fb_map drivers/video/console/fbcon.c
--- 25/drivers/video/console/fbcon.c~signed-bug-in-drivers-video-console-fbconc-con2fb_map	2004-06-28 13:32:20.390461248 -0700
+++ 25-akpm/drivers/video/console/fbcon.c	2004-06-28 13:32:20.396460336 -0700
@@ -101,7 +101,7 @@
 #endif
 
 struct display fb_display[MAX_NR_CONSOLES];
-char con2fb_map[MAX_NR_CONSOLES];
+signed char con2fb_map[MAX_NR_CONSOLES];
 static int logo_height;
 static int logo_lines;
 static int logo_shown = -1;
diff -puN drivers/video/console/fbcon.h~signed-bug-in-drivers-video-console-fbconc-con2fb_map drivers/video/console/fbcon.h
--- 25/drivers/video/console/fbcon.h~signed-bug-in-drivers-video-console-fbconc-con2fb_map	2004-06-28 13:32:20.391461096 -0700
+++ 25-akpm/drivers/video/console/fbcon.h	2004-06-28 13:32:20.396460336 -0700
@@ -36,7 +36,7 @@ struct display {
 };
 
 /* drivers/video/console/fbcon.c */
-extern char con2fb_map[MAX_NR_CONSOLES];
+extern signed char con2fb_map[MAX_NR_CONSOLES];
 extern int set_con2fb_map(int unit, int newidx);
 
     /*
_
